### ISSUE TITLE
Fix automatic builds on readthedocs

### ISSUE THREAD

**Issue by mgeier** [bloomberg/bqplot#1020]
According to https://readthedocs.org/projects/bqplot/builds/, the last build of "latest" was one year ago (Dec. 21, 2018).

Normally this happens each time something is pushed to `master`, maybe there's something wrong with the repo setup?

**Comment by mgeier** [comment]
This still doesn't seem to be resolved.

Is https://bqplot.readthedocs.io/ still the official documentation site?

If yes, it's very much outdated by now!

**Comment by SylvainCorlay** [comment]
Thanks, I missed this. Putting this on my todo for the next release. We certainly need to completely redo the docs configuration after the move to the bqplot organization.

**Comment by mgeier** [comment]
You don't really need to wait for a release, right?

BTW, you should also consider to activate this feature: https://docs.readthedocs.io/en/stable/guides/autobuild-docs-for-pull-requests.html

With this, the docs are built with each PR, which is very handy!

**Comment by SylvainCorlay** [comment]
Yes, I do this for most packages I work with! I meant the next release as a mental milestone :)
